src/Emitter.ts

```diff
@@ -845,7 +845,7 @@ export default class Emitter
 		if (this._emit)
 		{
 			//decrease spawn timer
-			this._spawnTimer -= delta;
+			this._spawnTimer -=  Math.max(delta, 0);
```

Comment:
I'd suggest `delta >= 0 ? delta : 0`, as the ternary operator is still faster than using `Math.max()`. It would also protect from a delta of `NaN`.

Reply:
I had a feeling you'd want that.

Reply:
I'm the type of person who is quite happy to use `x != x` all over the place instead of `isNaN(x)`.